### PR TITLE
added test for [MTB-PL2-BC-OPS-3]

### PR DESCRIPTION
**Pull request by phoenixking25**

### CODE REVIEW

**Comment by phoenixking25** [comment]
> Tenant administrators should be able to create their own `roles` and `rolebindings`, for namespaced respurces.
> 
> Besides checking for the RPAC privileges, we will need to check if a tenant administrator can actually create a role and a rolebinding to that new role.
> 
> I am thinking it makes sense to combine these in a single test. What do you think?

please take a look now.

File: benchmarks/e2e/tests/create_role_bindings/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Behavioral
+
+**Category:**
+
+Self-Service Operations
+
+**Description:**
+
+Tenants should be able to do self service by creating own roles and role-bindings in their namespaces by binding them together.
```
Comment (JimBugwadia) [inline benchmarks/e2e/tests/create_role_bindings/README.md]: change `do self service` to `perform self-service operations`

File: benchmarks/e2e/tests/create_role_bindings/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+**Audit:**
+
+Run the following commands to check for permissions to manage `rolebinding` and `role` for each verb(get, list, create, update, patch, watch, delete, and deletecollection) in the tenant namespace:
+
+    kubectl --kubeconfig=tenant-a -n a1 auth can-i <verb> <resource>
+
+Each command must return 'yes'
+
+Create a `role` and `rolebinding` in the tenant namespace as tenant-administrator for the `namespaced resources` and further bind the role with the help of newly created rolebinding. It should be a success.
```
Comment (JimBugwadia) [inline benchmarks/e2e/tests/create_role_bindings/README.md]: Would be good to provide more details, with examples (kubectl and YAML). For example:
- create a rolebinding to cluster role `edit`
- create a role `pod-view` with permission to view pods and create a role binding to this role

Comment (phoenixking25) [reply benchmarks/e2e/tests/create_role_bindings/README.md]: Understood

File: benchmarks/e2e/tests/create_role_bindings/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Behavioral
+
+**Category:**
+
+Self-Service Operations
+
+**Description:**
+
+Tenants should be able to perform self-service operations by creating own roles and role-bindings in their namespaces by binding them together.
```
Comment (JimBugwadia) [inline benchmarks/e2e/tests/create_role_bindings/README.md:17]: Suggest wording as
> Tenants should be able to create roles and role-bindings in their namespaces

File: benchmarks/e2e/tests/create_role_bindings/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Tenants
+
+**Rationale:**
+
+Enables self-service management of roles/role-bindings to bind allowed roles/tenant users.
```
Comment (JimBugwadia) [inline benchmarks/e2e/tests/create_role_bindings/README.md:23]: Suggested wording:
> Enables self-service management of roles and role-bindings within a tenant namespace

**Pull request merged**